components/brave_ads/browser/ads_tooltips_delegate.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_TOOLTIPS_DELEGATE_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_TOOLTIPS_DELEGATE_H_
+
+#include <string>
+
+#include "brave/components/brave_ads/browser/ads_tooltips_delegate.h"
```

Comment:
You're including the file as a header for itself

Reply:
Oops, removed.